Source/WebCore/workers/service/ServiceWorkerRegistration.cpp

```diff
@@ -352,8 +352,23 @@ void ServiceWorkerRegistration::getNotifications(const GetNotificationOptions& f
 CookieStoreManager& ServiceWorkerRegistration::cookies()
 {
     if (!m_cookieStoreManager)
-        m_cookieStoreManager = CookieStoreManager::create();
+        m_cookieStoreManager = CookieStoreManager::create(*this);
     return *m_cookieStoreManager;
 }
 
+void ServiceWorkerRegistration::addCookieChangeSubscriptions(Vector<CookieChangeSubscription>&& subscriptions, Ref<DeferredPromise>&& promise)
+{
+    m_container->addCookieChangeSubscriptions(identifier(), WTFMove(subscriptions), WTFMove(promise));
```

Comment:
Fixed! Since `isContextStopped()` also checks if the `ScriptExecutionContext` is not null, I removed the check for that and use only `isContextStopped()`